src/screens/userInput/UserInput.tsx

```diff
@@ -47,7 +47,8 @@ export const UserInput = ({ navigation }) => {
       type: 'viewer/setAccountId',
       payload: accountId,
     });
-    navigation.navigate(Routes.MultiView);
+    // navigation.navigate(Routes.MultiView);
+    navigation.navigate(Routes.SingleStreamView);
```

Comment:
shouldn't we navigate to SingleView only when connection is established already (from Multiview screen)?

Reply:
I have made the proposed changes as discussed with you.